src/biogeophys/WaterDiagnosticBulkType.F90

```diff
@@ -400,6 +405,13 @@ subroutine InitBulkHistory(this, bounds)
          avgflag='A', &
          long_name=this%info%lname('snow height of snow covered area'), &
          ptr_col=this%snow_depth_col, c2l_scale_type='urbanf')
+    this%snow_5day_col(begc:endc) = spval
```

Comment:
Should this be in an if use_cn statement?

Reply:
This could be within a use_cn statement as it's only used when use_cn is on as well as onset_thresh_depends_on_veg.

Reply:
I decided against adding a use_cn if statement. But, we should keep in mind that in many cases these accumulator variables are only used in certain configurations, but are often used all the time. That does allow them to be output to history files. Figuring out when accumulators need to be used would either require complex logic that would cause tendrils of usage beyond the modules that need it -- or a better mechanism to track this sort of thing seperately.